docker/dev-standalone/README.md

```diff
@@ -0,0 +1,96 @@
+# liftbridge-docker
```

Comment:
Think we should just archive the [liftbridge-docker](https://github.com/liftbridge-io/liftbridge-docker) repo then?